src/UI/Component/Symbol/Factory.php

```diff
@@ -89,21 +87,21 @@ public function icon(): Icon\Factory;
 	 *   usage:
 	 *       1: Glyphs MUST NOT be used in content titles.
 	 *       2: >
-	 *          Glyphs MUST be used for cross-sectional functionality such as mail for
+	 *          Glyphs MUST be used for cross-sectional functionality such as Mail for
 	 *          example and NOT for representing objects.
 	 *       3: >
 	 *          Glyphs SHOULD be used for very simple tasks that are repeated at
 	 *          many places throughout the system.
 	 *       4: >
-	 *          Services such as mail MAY be represented by a glyph AND an icon.
+	 *          Services such as Mail MAY be represented either by a glyph OR by an icon plus text label, depending on the usage scenario.
```

Comment:
Why is mail capitalised here? We only capitalise names of other UI Components in the description and rules, to make them stand out. Note, Glyph and Icon should be capitalised, since they are other UI Components, not your bad ;-)